nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -1145,9 +1145,12 @@ systemd.services.nginx.serviceConfig.User = lib.mkForce "root";
      As well as this, the options <literal>security.acme.acceptTerms</literal> and either
      <literal>security.acme.email</literal> or <literal>security.acme.certs.&lt;name&gt;.email</literal>
      must be set in order to use the ACME module.
-     Certificates will be regenerated anew on the next renewal date. The credentials for simp-le are
-     preserved and thus it is possible to roll back to previous versions without breaking certificate
-     generation.
+     Certificates will be regenerated on activation, no account or certificate will be migrated from simp-le.
```

Comment:
`simp_le` state and `lego` state are stored in different directories no? or at different places? So rollback _should_ work? if that isn't the case anymore than that's a new bug.

Reply:
We dont say it's backwards compatible. We just say that rollback is an option. Those are different things

Reply:
Ah yes, if that’s what you mean for "rollback". However:
- a new account will be recreated by lego anwyay, he won’t be able to use the one create by simp_le
- lego will generate new certificates when switching from simp-le to lego the first time, because it doesn’t use the content of /var/lib/acme/your_cert to make its decision. Switching back I don’t know (that will depend on how simp-le computes if a certificate is too old)

Reply:
We mean rollback in the sense of  `nixos-rebuild switch --rollback`.  i.e.  the switch to `lego` doesn't destroy any `simp_le` state so if something goes bad with upgrade, you can go back to `19.09` and figure out what went wrong

Reply:
Ok, I’ll put back the last line then

Reply:
Changed